operations/README.md

```diff
@@ -20,6 +20,11 @@ Role Variables
     # Docker version to update to.
     osbs_docker_version: <default not set>
 
+    # Update OpenShift node labels.
+    osbs_update_labels: false
```

Comment:
Since `labels` can be applied to any object in OpenShift, can this be `osbs_update_node_labels` ?

Reply:
sure